frontend/src/App.css

```
@@ -1000,158 +1000,33 @@ footer {
+.csmf-gridline { position: absolute; left: 0; right: 0; border-top: 1px dashed #eef2f6; }
+.csmf-bars { position: absolute; inset: 0; display: flex; align-items: flex-end; justify-content: space-around; padding: 0 4px; }
+.csmf-group { display: flex; align-items: flex-end; gap: 2px; height: 100%; }
+.csmf-bar { width: 12px; border-radius: 2px 2px 0 0; }
+.csmf-bar.uncal { background: #7da7d9; }
+.csmf-bar.cal { background: #a9ad6e; position: relative; }
+.csmf-whisker { position: absolute; left: 50%; transform: translateX(-50%); width: 2px; background: #3f3f46; }
+.csmf-whisker::before, .csmf-whisker::after { content: ''; position: absolute; left: -3px; width: 8px; height: 2px; background: #3f3f46; }
+.csmf-whisker::before { top: 0; }
+.csmf-whisker::after { bottom: 0; }
+.csmf-xlabels { display: flex; justify-content: space-around; margin-left: 30px; margin-top: 4px; }
+.csmf-xlabel { width: 28px; text-align: center; font-size: 9px; line-height: 1.1; color: #64748b; word-break: break-word; }
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)